src/lib/get-target-path.ts

```diff
@@ -6,6 +6,7 @@
 import {
 	TFE_VERSION_IN_PATH_REGEXP,
 	VERSION_IN_PATH_REGEX,
+	NO_V_VERSION_IN_PATH_REGEX,
```

Comment:
Let's add some tests to this file: https://github.com/hashicorp/dev-portal/blob/a0fe44e1125aea66e2b89238bfcfc238620c2937/src/lib/__tests__/get-target-path.test.ts.